packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.tsx

```diff
@@ -0,0 +1,142 @@
+// Styles
+import './VNavigationDrawer.sass'
+
+// Composables
+import { makeBorderProps, useBorder } from '@/composables/border'
+import { makeBorderRadiusProps, useBorderRadius } from '@/composables/border-radius'
+import { makeDimensionProps, useDimension } from '@/composables/dimensions'
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makeLayoutItemProps, useLayoutItem } from '@/composables/layout'
+import { makePositionProps, usePosition } from '@/composables/position'
+import { useProxiedModel } from '@/composables/proxiedModel'
+import { useTheme } from '@/composables/theme'
+
+// Utilities
+import { computed, defineComponent, onBeforeMount, ref, toRef } from 'vue'
+import { convertToUnit } from '@/util/helpers'
+import { makeTagProps } from '@/composables/tag'
+import makeProps from '@/util/makeProps'
+
+export default defineComponent({
+  name: 'VNavigationDrawer',
+
+  props: makeProps({
+    ...makeBorderProps(),
+    ...makeBorderRadiusProps(),
+    ...makeDimensionProps({ width: 256 }),
```

Comment:
Using this composable means it will get all of the width/height props but using them (except for width) won't have any effect because they will be (rightly) overwritten by layout styles